typed-builder-macro/src/field_info.rs

```diff
@@ -345,6 +378,58 @@ impl ApplyMeta for SetterSettings {
     }
 }
 
+#[derive(Debug, Clone)]
+pub enum StripOption {
+    Span(Span),
+    WithFallback(Span, String),
+}
+
+impl StripOption {
+    fn span(&self) -> &Span {
+        match self {
+            StripOption::Span(span) => span,
+            StripOption::WithFallback(span, _) => span,
+        }
+    }
+
+    pub fn from_args(sub: SubAttr) -> Result<Self, Error> {
+        let name = sub.name.clone();
+        let mut total = 0;
+        let mut result: Result<Self, Error> = Err(Error::new_spanned(
+            &name,
+            format!("Parameters required {:?}", name.to_string()),
+        ));
+
+        for arg in sub.args::<AttrArg>()? {
+            if total > 0 {
+                result = Err(Error::new_spanned(
+                    arg.name(),
+                    format!("Too many paramters {:?}", arg.name().to_string()),
+                ));
+
+                continue;
+            }
+
+            if arg.name().to_string().as_str() != "fallback" {
+                result = Err(Error::new_spanned(
```

Comment:
Nit: as I've said before that this entire thing should be replaced with the `ApplyMeta` style, but just want to point out that instead of setting `result` here and `continue`-ing, you could have just `return`ed the error from here.

This also applies to the previous `if` in this loop, but more so to this one because it creates a bug. Consider:
```rust
#[builder(setter(strip_option(invalid = setting, fallback = "outer_opt"))]
```

The first iteration will set `result` to `Err` for the `invalid = setting` - but then the next iteration will set it to `Ok` for the `fallback`.

Reply:
I've updated this, but the only problem is that the code now allows for:

- multiple sub attributes with the same name `strip_option(fallback = a, fallback = b)` 
- empty sub attributes `strip_option()`

If I update the `apply_sub_attr` in the `ApplyMeta` trait it might be a breaking change for code bases that may rely on this behaviour. I could override locally I suppose.

Is this important or should I ignore for now?